src/test/LogTest.cpp

```diff
@@ -0,0 +1,27 @@
+//
+// Created by XFC on 2018/6/24.
+//
+#include <iostream>
+#include <fstream>
+#include <thread>
+
+#include "../common/Log/Log.h"
+int main() {
+    snail::log::LogContext context;
```

Comment:
We needn't the namespace `Log`.
`snail` is enough.

Reply:
ok